sdk/identity/identity/samples/AzureCacheForRedis/node-redis.md

```diff
@@ -183,28 +183,40 @@ import * as dotenv from "dotenv";
 dotenv.config();
 
 async function returnPassword(credential: TokenCredential) {
-    // The scope will be changed for Azure AD Public Preview
-    const redisScope = "https://*.cacheinfra.windows.net:10225/appid/.default"
+    // The current scope is for public preview and may change for GA release.
```

Comment:
Comments like this should be removed. That way, if nothing changes for the GA, there's no more work to be done in this sample file.

Reply:
Got it. I think we'll need to do that for Java and Python samples too. @g2vinay

Reply:
we can update this for Java and Python.

Reply:
updated